Tasks/AzurePowerShellV3/AzurePowerShell.ps1

```diff
@@ -74,10 +78,27 @@ catch
 
 Update-PSModulePathForHostedAgent -targetAzurePs $targetAzurePs -authScheme $authScheme
 
+# troubleshoot link
+$troubleshoot = "https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops#troubleshooting"
+$status = $true
```

Comment:
you do not need to define this variable here. You can set it directly after the write-host statement.